report-viewer/src/components/ComparisonsTable.vue

```diff
@@ -2,121 +2,233 @@
-        comparison.similarity
-      "
-      class="selectable"
-    >
+  </table>
+
+  <RecycleScroller
+      style="height: 650px; overflow: auto;"
+      class="scroller"
+      :items="topComparisons"
+      :item-size="48"
+      key-field="id"
+      v-slot="{ item }"
+  >
+    <table class="table" style="width: 100%">
+      <tr v-if="item.id % 2 === 0" class="selectableEven">
       <td
         @click="
           navigateToComparisonView(
-            comparison.firstSubmissionId,
-            comparison.secondSubmissionId
+            item.firstSubmissionId,
+            item.secondSubmissionId
           )
         "
+        class="td1"
       >
-        {{ index + 1 }}.
+        {{ item.id }}.
       </td>
       <td
         @click="
           navigateToComparisonView(
-            comparison.firstSubmissionId,
-            comparison.secondSubmissionId
+            item.firstSubmissionId,
+            item.secondSubmissionId
           )
         "
         :class="{
-          'anonymous-style': isAnonymous(comparison.firstSubmissionId),
+          'anonymous-style': isAnonymous(item.firstSubmissionId),
         }"
+        class="td2"
       >
         {{
-          isAnonymous(comparison.firstSubmissionId)
+          isAnonymous(item.firstSubmissionId)
             ? "Hidden"
-            : displayName(comparison.firstSubmissionId)
+            : displayName(item.firstSubmissionId)
         }}
       </td>
       <td
         @click="
           navigateToComparisonView(
-            comparison.firstSubmissionId,
-            comparison.secondSubmissionId
+            item.firstSubmissionId,
+            item.secondSubmissionId
           )
         "
+        class="td3"
       >
         <img alt=">>" src="@/assets/double_arrow_black_18dp.svg" />
       </td>
       <td
         @click="
           navigateToComparisonView(
-            comparison.firstSubmissionId,
-            comparison.secondSubmissionId
+            item.firstSubmissionId,
+            item.secondSubmissionId
           )
         "
         :class="{
-          'anonymous-style': isAnonymous(comparison.secondSubmissionId),
+          'anonymous-style': isAnonymous(item.secondSubmissionId),
         }"
+        class="td4"
       >
         {{
-          isAnonymous(comparison.secondSubmissionId)
+          isAnonymous(item.secondSubmissionId)
             ? "Hidden"
-            : displayName(comparison.secondSubmissionId)
+            : displayName(item.secondSubmissionId)
         }}
       </td>
       <td
         @click="
           navigateToComparisonView(
-            comparison.firstSubmissionId,
-            comparison.secondSubmissionId
+            item.firstSubmissionId,
+            item.secondSubmissionId
           )
         "
+        class="td5"
       >
-        {{ formattedMatchPercentage(comparison.similarity) }}
+        {{ formattedMatchPercentage(item.similarity) }}
       </td>
-      <td>
+      <td style="width: 5%">
```

Comment:
Partially including style in the html is a bad smell, please move to the style part